Removed unnecessary refs to old edge versions from webdriver doc (#1435)
MicrosoftDocs/edge-developer: microsoft-edge/webdriver-chromium/capabilities-edge-options.md

---
# Capabilities and EdgeOptions  

> Capabilities are options that you may use to customize and configure an `EdgeDriver` session.  To learn about starting a new `EdgeDriver` session, navigate to [Automating Microsoft Edge][WebdriverIndexAutomateMicrosoftEdgeChromium].  This article describes all supported capabilities for Microsoft Edge and details on passing the capabilities to `EdgeDriver` sessions.  

Reviewer: ```suggestion
Capabilities are options that you can use to customize and configure an `EdgeDriver` session.  To learn about starting a new `EdgeDriver` session, navigate to [Automating Microsoft Edge][WebdriverIndexAutomateMicrosoftEdgeChromium].  This article describes all supported capabilities for Microsoft Edge and provides details about passing the capabilities to `EdgeDriver` sessions.  

```
